### PR TITLE
New notification type

### PR DESCRIPTION
**Pull request by sineade**
:elephant: :art:
Added a new notification for success messages for the Auto renew/mpp credit card management. 

screen shot 2017-03-09 at 11 45 03 am

### CODE REVIEW

File: app/helpers/ama_layout_content_helper.rb [hunk 1]
```diff
@@ -53,4 +53,8 @@ def notice(notice)
   def alert(alert)
     render "ama_layout/alert" if alert
   end
+
+  def success_message(success)
+    render "ama_layout/success" if success
```
Comment (MathieuGilbert) [inline app/helpers/ama_layout_content_helper.rb]: What do you think of passing the success message to the partial as a variable, so it doesn't haven't to read from the `flash` object?

`render partial: "ama_layout/success", locals: { message: success } if success`

File: app/views/ama_layout/_success.html.erb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<div class="row">
+  <div class="large-12">
+    <div class="success_notification"><span><%= flash[:success].html_safe %></span></div>
```
Comment (MathieuGilbert) [inline app/views/ama_layout/_success.html.erb]: The `flash[:success]` would get replaced with `message` (or whatever variable name you used in the "locals" assignment).

**Comment by MathieuGilbert** [comment]
💩

**Comment by MathieuGilbert** [comment]
`Coverage (99.50%) is below the expected minimum coverage (100.00%).`

Needs more tests.

**Comment by sineade** [comment]
@mvandenbeuken could you take a look at @MathieuGilbert 's comments please :)

File: app/helpers/ama_layout_content_helper.rb (hunk 1 above)
Comment (mvandenbeuken) [inline app/helpers/ama_layout_content_helper.rb]: great suggestion

**Comment by kaytcampbell** [comment]
:hankey: Bump Version.
I don't think it's in this PR but in your screenshot, automatic renewal is capitalized in the confirmation message, and it should not be.

**Comment by sineade** [comment]
the message was in membership so I fixed the capitalization there @kaytcampbell

**Comment by kathleenAMA** [comment]
Dont' we usually put the success messages in the page body?

**Comment by sineade** [comment]
@kathleenAMA yes i had to create a new success class to display it this way. It's how it was mocked up and approved though...

**Comment by kathleenAMA** [comment]
@sineade okay 👍

**Comment by kathleenAMA** [comment]
:art: :art: :art:

**Comment by jessedoyle** [comment]
:book:

**Comment by jessedoyle** [comment]
:elephant::elephant::elephant:

**Pull request merged**